core/base_abstractions/sensor.py

```diff
@@ -158,23 +166,34 @@ def _get_observation_space(self) -> gym.spaces.Tuple:
         only if the expert failed to generate a true expert action. The
         value `num actions in task` should be in `config["nactions"]`
         """
-        return gym.spaces.Tuple(
-            (gym.spaces.Discrete(self.nactions), gym.spaces.Discrete(2))
-        )
+        return su.flatten_space(self.unflattened_observation_space)
```

Comment:
Currently we have `def _get_observation_space(self) -> gym.spaces.Tuple:`, the returned space shouldn't be a Tuple after flattening right? Maybe I'm misunderstanding

Reply:
I probably left it like that after some initial modifications. Will set it to the current space.